admin_guide/cluster_management.rst

```diff
@@ -28,12 +28,105 @@ To add a new node to the cluster, you first need to add the DNS name or IP addre
 
    SELECT * from master_add_node('node-name', 5432);
 
-In addition to the above, if you want to move existing shards to the newly added worker, Citus Enterprise provides an additional rebalance_table_shards function to make this easier. This function will move the shards of the given table to make them evenly distributed among the workers.
+The new node is available for shards of new distributed tables. Existing shards will stay where they are unless redistributed, so adding a new worker may not help performance without further steps.
+
+Rebalancing Shards without Downtime
+-----------------------------------
+
+If you want to move existing shards to a newly added worker, Citus Enterprise and Citus Cloud provide a :ref:`rebalance_table_shards` function to make it easier. This function will move the shards of a given table to distribute them evenly among the workers.
 
 ::
 
-	select rebalance_table_shards('github_events');
+  SELECT rebalance_table_shards('github_events');
+
+Many products, like multi-tenant SaaS applications, cannot tolerate downtime, and Citus rebalancing is able to honor this requirement on PostgreSQL 10 or above. This means reads and writes from the application can continue without interruption while data is being moved.
```

Comment:
We don't quite describe the logic on how we cutover. Instead of "without interruption", should we say with "minimal interruption"? In the "How it works" section, could we also expand a bit on the process, and that we lock for only a short time during the cutover?